src/core/lib/iomgr/parse_address.cc

```diff
@@ -53,7 +53,11 @@ bool grpc_parse_unix(const grpc_uri* uri,
       reinterpret_cast<struct sockaddr_un*>(resolved_addr->addr);
   const size_t maxlen = sizeof(un->sun_path);
   const size_t path_len = strnlen(uri->path, maxlen);
-  if (path_len == maxlen) return false;
+  if (path_len == maxlen) {
+    gpr_log(GPR_ERROR, "Path name should not have more than %zu characters",
```

Comment:
For portability, I think you need to use the `PRIuPTR` macro here.

Reply:
outdated